crip/preprocess.py

```diff
@@ -78,7 +78,7 @@ def projectionsToSinograms(projections):
         Permute projections to sinograms by axes swapping `(views, h, w) -> (h, views, w)`.
     """
     (views, h, w) = projections.shape
-    sinograms = np.zeros((h, views, w))
+    sinograms = np.zeros((h, views, w), dtype=np.float32)
```

Comment:
Dtype will change if you do so. Like, users using np.float64, and we narrow it to float32.

Reply:
Maybe `zeros_like` is a better idea to keep dtype.